src/lib/runChecks.js

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License"); 
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS 
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const Mocha = require('mocha')
+const { logger } = require('./logger')
+
+// import individual checks
+const checkNodeVersion = require('../checks/checkNodeVersion')
+const descriptionNotEmpty = require('../checks/descriptionNotEmpty')
+const containsKeyword = require('../checks/containsKeyword')
+
+// array of checks that need to be run on metadata
+const checks = [descriptionNotEmpty, containsKeyword, checkNodeVersion]
+
+async function runChecks(metadataJSON) {
+    const mocha = new Mocha({
+        reporter: 'spec',
+    })
+    const suite = new Mocha.Suite('Validating App Builder templates')
+    suite.timeout(0)
```

Comment:
Why do we need timeout here? If we have any reason for it, please add the comment to the code

Reply:
This disables the timeout.